### PR TITLE
Path name resource entity added

### PR DESCRIPTION
**Pull request by Yumfriez**

### CODE REVIEW

File: src/main/java/com/epam/ta/reportportal/ws/model/item/LaunchPathName.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package com.epam.ta.reportportal.ws.model.item;
```
Comment (avarabyeu) [inline src/main/java/com/epam/ta/reportportal/ws/model/item/LaunchPathName.java]: header?

File: src/main/java/com/epam/ta/reportportal/ws/model/item/PathNameResource.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
+package com.epam.ta.reportportal.ws.model.item;
```
Comment (avarabyeu) [inline src/main/java/com/epam/ta/reportportal/ws/model/item/PathNameResource.java]: header?

File: src/main/java/com/epam/ta/reportportal/ws/model/item/LaunchPathName.java (hunk 1 above)
Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/ws/model/item/LaunchPathName.java]: added

File: src/main/java/com/epam/ta/reportportal/ws/model/item/PathNameResource.java (hunk 2 above)
Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/ws/model/item/PathNameResource.java]: added

**Pull request merged**